### DISCUSSION TITLE
How do i go about converting large (600mb+) models with command line tools?

### DISCUSSION THREAD

**Discussion by hl2jeep** [assimp/assimp#5761] [Q&A]
I'm trying to convert some large obj files into glb due to have some major texture issues that can't be resolved any other way. assimp seems to do the trick however it is abysmally slow. i ran yesterday a model overnight to convert and by next morning it was still going. i was using the outdated 5.2.5 sdk tools, but i can't figure out how to build a newer release, so far i've only been able to build dll's which in my usecase are useless. any ideas how i can do this without it taking countless hours?

**Comment by kimkulling** [comment]
Do you have some more content? Are the texture files available?

**Comment by hl2jeep** [comment]
Due to licensing I can't share anything, but the model has around 120
unique textures and 3 million vertices
